cmd/kube-controller-manager/app/core.go

```diff
@@ -265,7 +265,7 @@ func startPersistentVolumeBinderController(ctx context.Context, controllerContex
 		EnableDynamicProvisioning: controllerContext.ComponentConfig.PersistentVolumeBinderController.VolumeConfiguration.EnableDynamicProvisioning,
 		FilteredDialOptions:       filteredDialOptions,
 	}
-	volumeController, volumeControllerErr := persistentvolumecontroller.NewController(params)
+	volumeController, volumeControllerErr := persistentvolumecontroller.NewController(ctx, params)
```

Comment:
Add a name here as in https://github.com/kubernetes/kubernetes/pull/113315

Same for the other controllers in this PR.

Reply:
Still open?

Reply:
> Still open?

There are many changes. still in progress

Reply:
done